### PR TITLE
Solves #133: add client_name, logo_uri and contacts fields to client …

### PR DESCRIPTION
**Pull request by christophehenry**
Solves #133: add client_name, logo_uri and contacts fields to client regitration request

### CODE REVIEW

**Comment by jaxoncreed** [comment]
@dmitrizagidulin could you leave a review.

File: README.md [hunk 1]
```diff
@@ -155,6 +155,31 @@ and emits the following events:
 - `logout ()` when a user logs out
 - `session (session: Session | null)` when a user logs in or out
 
+### Client registration
+
+`SolidAuthClient` automatically registers your OIDC client application if it is
+unknown to the autorisation server, following
```
Comment (TallTed) [inline README.md]:
```suggestion
unknown to the authorization server, following
```

**Comment by jaxoncreed** [comment]
Pinging @dmitrizagidulin again

File: README.md [hunk 2]
```diff
@@ -155,6 +155,31 @@ and emits the following events:
...
+[the registration request spec](https://openid.net/specs/openid-connect-registration-1_0.html#RegistrationRequest).
+
+You can specify some fields of this registration request by passing them to the
+`loginSession` parameter of `solid.auth.login`.
+
+Supported fields are:
+
+*  `client_name` and internationalized variants (`clientName` property)
+* `contacts` (`contacts` property)
+* `logo_uri` (`contacts` property)
```
Comment (dmitrizagidulin) [inline README.md]:
```suggestion
* `logo_uri` (`logo_uri` property)
```

**Pull request merged**